app/src/sharedTest/java/org/oppia/app/utility/EspressoTestsMatchers.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.app.utility
+
+import android.view.View
+import org.hamcrest.Matcher
+
+// https://medium.com/@dbottillo/android-ui-test-espresso-matcher-for-imageview-1a28c832626f#.4snjg8frw
+/** This object mainly facilitates as bridge between test-cases and various custom test-matchers. */
+object EspressoTestsMatchers {
+
+  fun withDrawable(resourceId: Int): Matcher<View> {
+    return DrawableMatcher(resourceId)
+  }
+
+  @Suppress("unused")
+  fun noDrawable(): Matcher<View> {
+    return DrawableMatcher(-1)
```

Comment:
I suggest we do one of the following:
- Expose the ANY/EMPTY as constants for this class to consume rather than duplicating the '-1' sentinel value here
- Make DrawableMatcher's constructor private and use a factor to create three different versions: createMatcherForAnyDrawable(), createMatcherForNoDrawable(), createDrawableMatcherForResourceId()

I prefer the second option for readability, but both seem like sensible options.

Reply:
Tried doing the second option but was not able to finish this optimally, so moved to option one for now to move ahead with the merge.